lib/CPU/CPU.cpp

```diff
@@ -36,12 +36,35 @@
  * Registers
  */
 
-uint16_t CPU::AF = 0x01B0;    // AAAAAAAAZNHCxxxx
-uint16_t CPU::BC = 0x0013;    // BBBBBBBBCCCCCCCC
-uint16_t CPU::DE = 0x00D8;    // DDDDDDDDEEEEEEEE
-uint16_t CPU::HL = 0x014D;    // HHHHHHHHLLLLLLLL
-uint16_t CPU::SP = 0xFFFE;    // Stack Pointer
-uint16_t CPU::PC = PC_START;  // Program Counter
+// Accumulator and Flag Register
+// Bit 0 - 3:   Unused
+// Bit 4:       Carry Flag
+// Bit 5:       Half Carry Flag
+// Bit 6:       Add/Sub Flag
+// Bit 7:       Zero Flag
+// Bit 8 - 15:  Accumulator
+uint16_t CPU::AF = 0x01B0;
+
+// BC Register
+// Bit 0 - 7:   Gen. Purpose C Register
+// Bit 8 - 15:  Gen. Purpose B Register
+uint16_t CPU::BC = 0x0013;
+
+// DE Register
+// Bit 0 - 7:   Gen Purpose E Register
+// Bit 8 - 15:  Gen Purpose D Register
+uint16_t CPU::DE = 0x00D8; 
+
+// HL Register
+// Bit 0 - 7:   Gen Purpose L Register
+// Bit 8 - 15:  Gen Purpise H Register
+uint16_t CPU::HL = 0x014D;
+
+// Stack Pointer
+uint16_t CPU::SP = 0xFFFE;
+
+// Program Counter
+uint16_t CPU::PC = PC_START;
```

Comment:
Thanks for improving the documentation, I appreciate it!

I'm thinking about changing the registers into union types as I did with the APU registers but that's definitely for another time! First I wanna get this one merged.